### DISCUSSION TITLE
Use double brackets as markdown link

### DISCUSSION THREAD

**Discussion by danymat** [vuepress/core#203] [Q&A] [answered]
I solely use the Wikilinks syntax in order to link my notes.
I would love to use vuepress but I didn't manage to use this syntax for my links. Is it possible to do so?

**Comment by Mister-Hope** [comment, marked as answer]
We are using markdown-it to render markdown.

Search if there is a corresponding markdown-it plugin.

If not, you could write one yourself.

Just use the `extendsMarkfown` in plugin api to let markdown-it be able to parse the new syntax with this plugin.
